src/openms/include/OpenMS/METADATA/PeptideIdentification.h

```diff
@@ -38,11 +38,13 @@
 #include <OpenMS/METADATA/MetaInfoInterface.h>
 #include <OpenMS/METADATA/ProteinHit.h>
 
+
 #include <string>
 #include <map>
 
 namespace OpenMS
 {
+    class ConsensusMap;
```

Comment:
@cbielow I would have loved to resolve that discussion first

Reply:
I figured this to be of low importance, since this can be easily moved to some other places (where?), but here has the additional benefit of being visible as member function (global functions are notoriously hard to find, especially for non-core developers).
Can we think of some better solution? I'm happy to move the code.